MULTIPLATFORM.md

```diff
@@ -127,8 +127,20 @@ Run native macos Arm64 sample:
 ```bash
 ./gradlew :compose:mpp:demo:runDebugExecutableMacosArm64
 ```
-
-#### Run mpp/demo-uikit sample on iOS
+### Run in KMP Wizard project
+- To use a locally built compose in [KMP with Compose wizard project](https://kmp.jetbrains.com) you need to perform some extra steps:-Checkout https://github.com/JetBrains/compose-multiplatform
+- Open `gradle-plugins` in `compose-multiplatform`
+- Update `gradle.properties` by setting `compose.version` and `deploy.version` to the version you've published (`0.1.0-dev1000` in example above).
```

Comment:
It's also possible to publish it to mavenLocal without opening the project in IDE (in case it's easier and no changes in the code required):

```
cd compose-multiplatform/gradle-plugins
./gradlew publishToMavenLocal -Pcompose.version=0.1.0-dev1000 -Pdeploy.version=0.1.0-dev1000
```

Same for components, but with one more property: `-Pcompose.userMavenLocal=true`

Reply:
Open was meant to indicate the current working folder.
I'll just bring a script in a separate PR!